extensions/eclipse-che-theia-plugin-ext/src/browser/che-openshift-main.ts

```diff
@@ -24,13 +27,25 @@ export class CheOpenshiftMainImpl implements CheOpenshiftMain {
                 this.isMultiUser = true;
             }
         });
+        container.get<EnvVariablesServer>(EnvVariablesServer).getValue('CHE_API').then(variable => {
```

Comment:
how about getting `EnvVariablesServer` once and store it in a variable?
I believe inversifyjs is fast enough ) but less code duplication, at least

Reply:
fixed